src/components/ClassicForm/helpers.ts

```diff
@@ -154,9 +156,36 @@ export const getLogic = (logic: LogicChoice): LogicChoice => {
  * getAuthor('foo/nbar', 'and'); // author:("foo" "bar")
  * getAuthor('-foo/nbar', 'or'); // author:(-"foo" OR "bar")
  * getAuthor('foo$', 'and'); // author:"foo" author_count:1
+ * getAuthor('^foo', 'and'); // first_author:("foo") author_count:1
  */
-export const getAuthor = (author: string, logic: LogicChoice) =>
-  unless(
+export const getAuthor = (author: string, logic: LogicChoice) => {
+  const parseAndFormat = (list: Array<string>): string => {
+    const [firstAuthors, otherAuthors] = partition(startsWith('^'), list);
+    const firstAuthorResult = isEmpty(firstAuthors)
+      ? undefined
+      : pipe(
+          map(replace(/^\^/, '')),
+          parsePrefixesAndWrapItems,
+          joinItemsWithLogic(logic),
+          wrapWithField('first_author'),
+        )(firstAuthors);
+
+    const otherAuthorResult = pipe(
```

Comment:
Might need to check `isEmpty` here too.

Search `^huchra, john` generated this query: collection:(astronomy) author:() first_author:("huchra, john")

Getting error with this query because of the empty author list

Reply:
Good eye! I think I missed that one.  I'll fix and merge